src/test/java/dev/blaauwendraad/masker/json/JsonMaskerTestUtil.java

```diff
@@ -49,7 +43,13 @@ private static void applyConfig(JsonNode jsonMaskingConfig, JsonMaskingConfig.Bu
             JsonNode value = e.getValue();
             switch (key) {
                 case "maskKeys" -> builder.maskKeys(asSet(value, JsonNode::asText));
-                case "maskJsonPaths" -> builder.maskJsonPaths(asSet(value, JsonNode::asText));
+                case "maskJsonPaths" -> StreamSupport.stream(value.spliterator(), false).forEach(node -> {
```

Comment:
Nice, can you do the same for maskKeys? Should be the same logic

Reply:
Done